03-transactions.md

```diff
@@ -315,28 +394,32 @@ Thus, a simplified formula for *expected weight* is used, which assumes:
 * Signatures are 73 bytes long (the maximum length).
 * There are a small number of outputs (thus 1 byte to count them).
 * There are always both a `to_local` output and a `to_remote` output.
+* (if `option_anchor_outputs`) there are always both a `to_local_anchor` and `to_remote_anchor` output.
 
 This yields the following *expected weights* (details of the computation in [Appendix A](#appendix-a-expected-weights)):
 
-    Commitment weight:   724 + 172 * num-untrimmed-htlc-outputs
-    HTLC-timeout weight: 663
-    HTLC-success weight: 703
+    Commitment weight (no option_anchor_outputs):   724 + 172 * num-untrimmed-htlc-outputs
+    Commitment weight (option_anchor_outputs):     1124 + 172 * num-untrimmed-htlc-outputs
```

Comment:
I'm getting `1116 + 172 * num-untrimmed-htlc-outputs` (see my comment on the calculation details below).